src/components/review-cards.jsx

```diff
@@ -1,11 +1,85 @@
 import React from 'react';
 
 class ReviewCards extends React.Component{
-  render(){
-    return (
-      <h1>Review Cards</h1>
-    )
+  constructor(props){
+    super(props);
+    this.state = {
+      isQuestion: true,
+    }
+    this.nextCard = this.nextCard.bind(this);
+    this.lastCard = this.lastCard.bind(this);
+    this.changeStatus = this.changeStatus.bind(this);
   }
+
+  componentDidMount(){
+    this.props.setActiveCardToReviewCards(0);
+  }
+
+  nextCard(){
+    let curIndex = this.props.activeCardToReviewCards.id;
+    if(curIndex === this.props.cardsArray.length - 1) {
+      curIndex = 0;
+    }
+    else {
+      curIndex++;
+    }
+    this.props.setActiveCardToReviewCards(curIndex);
+    }
+
+    lastCard(){
+      let curIndex = this.props.activeCardToReviewCards.id;
+      if(curIndex == 0){
+        curIndex = this.props.cardsArray.length - 1;
+      }
+      else {
+        curIndex--;
+      }
+      this.props.setActiveCardToReviewCards(curIndex);
+    }
+
+    changeStatus(){
+      this.setState({
+        isQuestion: !this.state.isQuestion
+      });
+      console.log("currentStatus",this.state.isQuestion);
```

Comment:
Remember a `console.log` right after state won't represent the actual value of state usually. If you want to know the current value of a property in state, put the `console.log` in the `render` method.